src/Components/ComparePublicationData/CompareFormCategory.tsx

```diff
@@ -16,10 +16,10 @@ import { isLegalCategory, LegalCategoryOptions } from '../../utils/categoryUtils
 
 interface CompareFormCategoryProps {
   importPublication: ImportPublication;
-  isDuplicate: boolean;
+  isCristinPublicationSelected: boolean;
```

Comment:
usikker på navngivning. hva forklarer parameteren best ? "isduplicate" blir brukt både for situasjonen der man drar opp en Cristinpost og sammenligner OG for forfatter og instutisjons-duplikater